frontendUiTable/src/main/java/com/virtuslab/gitmachete/frontend/ui/cell/BranchOrCommitCellRenderer.java

```diff
@@ -154,17 +154,17 @@ private int calculateTextPadding(@NonNegative int maxPosition) {
     }
 
     @NonNegative
-    private int getMaxGraphNodePositionInRow(IGraphNode node) {
-      // If node is a child (non root) branch, then the text must be shifted right to make place
+    private int getMaxGraphNodePositionInRow(IGraphItem graphItem) {
+      // If item is a child (non root) branch, then the text must be shifted right to make place
       // for the corresponding the right edge to the left.
-      // If node is a commit, then the text must be shifted right to keep it horizontally aligned
-      // with the corresponding branch node.
-      boolean isRootBranch = node.isBranch() && ((BranchNode) node).getBranch().isRootBranch();
-      return node.getIndentLevel() + (isRootBranch ? 0 : 1);
+      // If item is a commit, then the text must be shifted right to keep it horizontally aligned
+      // with the corresponding branch item.
+      boolean isRootBranch = graphItem.isBranch() && ((BranchItem) graphItem).getBranch().isRootBranch();
```

Comment:
Maybe add `asBranch`/`asCommit` methods (like with `asRootBranch` in `com.virtuslab.gitmachete.backend.api.BaseGitMacheteBranch`) instead of downcasts